account_vat_period_end_statement/views/account_view.xml

```diff
@@ -0,0 +1,169 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  Copyright 2019 Simone Rubino - Agile Business Group
+  License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+  -->
+<odoo>
+    <!--    Statement   -->
+
+    <record id="view_account_vat_period_end_statement" model="ir.ui.view">
+        <field name="name">account.vat.period.end.statement.form</field>
+        <field name="model">account.vat.period.end.statement</field>
+        <field name="type">form</field>
+        <field name="arch" type="xml">
+            <form string="Vat statement">
+                <header>
+                    <button name="create_move" states="draft" type="object" string="Create Move" class="oe_highlight"/>
+                    <button name="statement_draft" type="object" states="confirmed,paid" string="Set to draft"/>
+                    <button type="object" name="compute_amounts" string="Recompute amounts" states="draft"/>
+                    <field name="state" widget="statusbar"/>
+                </header>
+                <group>
+                    <field name="journal_id"   widget="selection" select="1"/>
```

Comment:
Questi `select="1"` sarebbero da rimuovere, non essendo più supportati.

L'ideale sarebbe creare la search view

Reply:
@eLBati come mai dici che non è supportato?
Onestamente non so nemmeno cosa faccia, ho cercato nella documentazione e nel codice ma non lo trovo (`select` è un termine troppo comune e trovo di tutto).

Reply:
Perchè non fa quello per cui è nato: rendere ricercabile il campo nella search view

Reply:
@eLBati fatto in https://github.com/OCA/l10n-italy/compare/cbb5f75b7150dd748a52e2d338c6dc895d3a2286..38379f047bfaa2dbb2805a7af4ed9962e8bef6bf, grazie